sql/core/src/main/scala/org/apache/spark/sql/execution/SQLExecution.scala

```diff
@@ -87,6 +87,11 @@ object SQLExecution {
       val planDescriptionMode =
         ExplainMode.fromString(sparkSession.sessionState.conf.uiExplainMode)
 
+      val globalConfigs = sparkSession.sharedState.conf.getAll.toMap
+      val modifiedConfigs = sparkSession.sessionState.conf.getAllConfs
+        .filterNot(kv => globalConfigs.get(kv._1).contains(kv._2))
```

Comment:
How much time will be spent on getting the modifiedConfigs here?

Reply:
Beanmark locally, its quite fast (0.0x ms) since the filter option just cost `O(N)`.